src/utilities/html.py

```diff
@@ -13,3 +15,42 @@ def textify(html):
     # don't ignore links anymore, I like links
     h.ignore_links = False
     return h.handle(html)
+
+
+LIST_PREFIX_RE = re.compile(r'^([0-9]+|[a-zA-Z])\.(?=[^\s])')
```

Comment:
When adding regex, please provide a link to an explanation of the code, or code comment that explains it, and what the purpose of it is / what we need it for.  Give examples.  For example a link to a site like: https://regex101.com/r/402DAE/1 that inclides the code and some example/test strings.